eng/SetupVSHive.ps1

```diff
@@ -9,7 +9,7 @@ $vsDir = $vsInfo.installationPath.TrimEnd("\")
 
 $vsRegEdit = Join-Path (Join-Path (Join-Path $vsDir 'Common7') 'IDE') 'VSRegEdit.exe'
 
-$hive = "RoslynDev"
+$hive = "RazorDev"
```

Comment:
Why this change? Surely the CI machines start clean, and don't care what the hive name is?